library/core/src/mem/manually_drop.rs

```diff
@@ -148,8 +147,8 @@ impl<T: ?Sized> ManuallyDrop<T> {
     /// This is normally prevented by the type system, but users of `ManuallyDrop` must
     /// uphold those guarantees without assistance from the compiler.
     ///
-    /// [`ManuallyDrop::into_inner`]: #method.into_inner
-    /// [`ptr::drop_in_place`]: ../ptr/fn.drop_in_place.html
+    /// [`into_inner`]: ManuallyDrop::into_inner
+    /// [`drop_in_place`]: ptr::drop_in_place
```

Comment:
Is `ptr` in scope in this module? If so this is fine, but otherwise it needs to be `crate::ptr` or `super::ptr`.

Reply:
I thought that `ptr` was in scope since it doesn't give me `results in unresolved link` error... but anyway I will add `crate`. Thanks